sevntu-checks/src/main/java/com/github/sevntu/checkstyle/checks/coding/CustomDeclarationOrderCheck.java

```diff
@@ -53,7 +53,7 @@
  * insensitive):
  * <ol>
  * <li>"Field" to denote the Fields</li>
- * <li>"DeclareAnnonClassField" to denote the fields keeping objects of anonymous classes</li>
+ * <li>"DeclareAnonClassField" to denote the fields keeping objects of anonymous classes</li>
```

Comment:
This is an API change.

My view is that, yes, someone could suffer from the transition between an old version of the checker and a new version, but they'll update their codebase, and cascade the fix through.

Reply:
@jsoref , please move this case to separate issue. It is ok to make such change in this project.

Reply:
I have this queued as a distinct commit. I'll send it later.

Reply:
Please pay attention - we need ISSUE. To make well visible/declared to users. It will be used in release notes as braking compatibility note.

Reply:
Oh, sure.